MSRV build is broken

Currently, bumpalo does not build with its current MSRV 1.56.0 anymore, as can be seen here: https://github.com/fitzgen/bumpalo/actions/runs/4247510260/jobs/7385640541

> error: package `csv v1.2.0` cannot be built because it requires rustc 1.60 or newer, while the currently active rustc version is 1.56.0
Error: Process completed with exit code 101.

The `csv` crate changed its MSRV to 1.60 as part of the csv 1.2.0 release ([corresponding commit](https://github.com/BurntSushi/rust-csv/commit/521294f87b3e1e286829f85a34f397e007f081ed)).

So I guess the MSRV of `bumpalo` has to be increased, too.

--- a/.github/workflows/rust.yaml
+++ b/.github/workflows/rust.yaml
@@ -14,7 +14,7 @@ jobs:
   build:
     strategy:
       matrix:
-        rust_channel: ["stable", "beta", "nightly", "1.56.0"]
+        rust_channel: ["stable", "beta", "nightly", "1.60.0"]
         feature_set: ["--features collections,boxed"]
         include:
           - rust_channel: "nightly"

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -8,6 +8,7 @@ Released YYYY-MM-DD.
 
 ### Changed
 
+* The minimum supported Rust version (MSRV) is now 1.60.0.
 * TODO (or remove section if none)
 
 ### Deprecated

--- a/README.md
+++ b/README.md
@@ -209,7 +209,7 @@ v.push(2);
 
 #### Minimum Supported Rust Version (MSRV)
 
-This crate is guaranteed to compile on stable Rust **1.56** and up. It might
+This crate is guaranteed to compile on stable Rust **1.60** and up. It might
 compile with older versions but that may change in any new patch release.
 
 We reserve the right to increment the MSRV on minor releases, however we will

